### PR TITLE
Add oidc group claim in config-auth、member、administrator

### LINKED ISSUE #9024: Allow admin setting group claim name to UI
The should be a item added to the UI as parameter of OIDC authentication.

The label "Group Claim Name"
Tool tip: "The name of Claim in the ID token whose value is the list of group names"

It should be a string with length no longer than 256 chars, no space is allowed.

### LINKED ISSUE #9026: Enable adding group to project when the auth backend is OIDC

### PR DESCRIPTION
**Pull request by jwangyangls**
1. add oidc group claim in configration page
2. add oidc group-page in user page,you can go to it through clicking add-group button;
3. add oidc list page in administrator ,admin can go to this page

Signed-off-by: Yogi_Wang <yawang@vmware.com>
image

image

image

image

image
fix #9024 ;
fix #9026 ;

### CODE REVIEW

File: src/portal/src/app/base/harbor-shell/harbor-shell.component.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
                             <clr-icon shape="users" clrVerticalNavIcon></clr-icon>
                             {{'SIDE_NAV.SYSTEM_MGMT.USER' | translate}}
                         </a>
-                        <a *ngIf='isLdapMode' clrVerticalNavLink routerLink="/harbor/groups" routerLinkActive="active">
+                        <a *ngIf='isLdapMode || isOidcMode' clrVerticalNavLink routerLink="/harbor/groups" routerLinkActive="active">
```
Comment (reasonerjt) [inline src/portal/src/app/base/harbor-shell/harbor-shell.component.html]: why http mode does not have this?

Comment (jwangyangls) [reply src/portal/src/app/base/harbor-shell/harbor-shell.component.html]: yes, it should have http mode, let me check why not.

Comment (jwangyangls) [reply src/portal/src/app/base/harbor-shell/harbor-shell.component.html]: Oh I found that I made a wrong change in an add white list commit,I correct it.
commit 274ab7bcffaf48d5d70458a21cc2b18639787c53

**Comment by rafael-fecha** [comment]
in which version is this new feature provided ?

**Pull request merged**